common/test/acceptance/pages/studio/users.py

```diff
@@ -78,6 +80,48 @@ def click_submit_new_user_form(self):
         self.q(css='.form-create.create-user .action-primary').click()
         wait_for_ajax_or_reload(self.browser)
 
+    def get_user(self, email):
+        """ Gets user wrapper by email """
+        target_users = [user for user in self.users if user.email == email]
+        assert len(target_users) == 1
+        return target_users[0]
+
+    def add_user_to_course(self, email):
+        """ Adds user to a course/library """
+        self.click_add_button()
+        self.wait_for(lambda: self.new_user_form_visible, "Add user form is visible")
+        self.set_new_user_email(email)
+        self.click_submit_new_user_form()
+
+    def delete_user_from_course(self, email):
+        """ Deletes user from course/library """
+        target_user = self.get_user(email)
+        target_user.click_delete()
+
+    def modal_dialog_visible(self, dialog_type):
+        """ Checks if modal dialog of specified class is displayed """
+        return self.q(css='.prompt.{dialog_type}'.format(dialog_type=dialog_type)).visible
+
+    def modal_dialog_text(self, dialog_type):
+        """ Gets modal dialog text """
+        return self.q(css='.prompt.{dialog_type} .message'.format(dialog_type=dialog_type)).text[0]
+
+
+class UsersPage(UsersPageMixin):
```

Comment:
I don't think you need this separate UsersPage class anymore (it is only used by LibraryUsersPage).